packages/mobile/react-native.config.js

```diff
@@ -7,4 +7,11 @@ module.exports = {
       },
     },
   },
+  dependencies: {
+    'react-native-fast-crypto': {
+      platforms: {
```

Comment:
What does this do?

Reply:
It disables `react-native-fast-crypto` for iOS

Reply:
Why would we want to do that?

Reply:
@cmcewen I disagree that we shouldn't be using rn-fast-crypto on ios. The lib seems to support ios. 
@i1skn if fixing it would be a lot of work, it's fine to do it in a separate PR but you should leave a TODO here and open an issue

Reply:
Well, `react-native-fast-crypto` and `flipper` relies on conflicting libraries. I have not checked how hard it would be to fix `react-native-fast-crypto`, but I can do it if having disabled for iOS would be a significant impact onto UX. Another option is to not have Flipper on iOS until we find a better solution. So, @cmcewen @jmrossy what do you think?

Reply:
Sorry @i1skn I wasn't clear - I meant that android was the reason we added this library, not that we could turn it off on iOS. I believe bip39 still relies on it and I think it would potentially break some features. I think it should not be that hard to only include on copy of those .a files, no?

Reply:
Gotcha, I will investigate then :)